src/state_circuit/memory.rs

```diff
@@ -368,6 +385,7 @@ mod tests {
         let op_1 = MemoryOp {
             address: MemoryAddress(pallas::Base::one()),
             steps: vec![
+                Some(ReadWrite::Write(Step(0), Value(pallas::Base::from_u64(0)))),
```

Comment:
We don't have to explicitly provide the init case in the witness; the `init()` function takes care of it for us.